src/portable/nordic/nrf5x/dcd_nrf5x.c

```diff
@@ -253,13 +244,27 @@ void dcd_init (uint8_t rhport)
 void dcd_int_enable(uint8_t rhport)
 {
   (void) rhport;
+#if defined(SOFTDEVICE_PRESENT)  &&  defined(OLD_NORDIC_SDK)
+  if (sd_nvic_EnableIRQ(USBD_IRQn) != NRF_SUCCESS)
+  {
+    NVIC_EnableIRQ(USBD_IRQn);
+  }
+#else
```

Comment:
do we really need it this, USB isn't restricted peripheral by SoftDevice ? If it is required to use sd_nvic_EnableIRQ() when sd is enabled, we should use this instead

```
#ifdef SOFTDEVICE_PRESENT
  if ( is_sd_enabled() ) sd_nvic_EnableIRQ(USBD_IRQn)
```

Reply:
I'm actually not absolutely sure if this is required.  This is more a defensive approach.  If one guarantees, that "NVIC_EnableIRQ(USBD_IRQn)" works for all Nordic SDK, I'll remove those #ifdefs immedately.  Same is true, if you accept the PR only if this construct is removed of course ;-)

is_sd_enabled() is not available in all Nordic SDKs.  Before using another #ifdef I prefer a "generic" approach which works for all Nordic SDKs.